### ISSUE TITLE
Ice crystal server bug

### ISSUE THREAD

**Issue by MrJorman** [BobMowzie/MowziesMobs#577]
![image](https://user-images.githubusercontent.com/55187962/151563523-3932f997-7a88-43e0-b181-50f449a5245f.png)
my players use ice_crystal.. and my and players screen....

**Comment by BobMowzie** [comment]
Sorry, I don’t understand what you mean…

**Comment by MrJorman** [comment]
When I use an ice crystal, then all the players that are next to me on the screen have this thing, which is higher in the screenshot.

**Comment by BobMowzie** [comment]
What is that thing? Does it appear when you use any other Mowzie’s Mobs items? What other mods do you have installed?

**Comment by MrJorman** [comment]
no, only on an ice crystal. I use Optifine, Twilight Forest, Abnormals mods.

**Comment by BobMowzie** [comment]
I can tell from the screenshot that you have other mods. I need to see a full mods list.

**Comment by MrJorman** [comment]
![image](https://user-images.githubusercontent.com/55187962/151641445-1af439b0-a7a3-4729-b0e0-9c8ec197f0f3.png)
![image](https://user-images.githubusercontent.com/55187962/151641449-248596ca-49f2-49ec-96ba-7fd920f78d6e.png)
![image](https://user-images.githubusercontent.com/55187962/151641451-d9cfc98e-e33e-45b0-8461-d08665a39692.png)

**Comment by BobMowzie** [comment]
Do any of these mods display the third person player model while you are in first person? Like, if you look down, can you see your legs?

**Comment by MrJorman** [comment]
No. Please note that there was no such thing on versions mowziesmobs-1.5.19.

**Comment by MrJorman** [comment]
++++1 bug in the piggy bank, I found 1 more bug, if you take an axe of a thousand metals and cut wood, after that he will not be able to use his skill on the PCM, you need to change the axe slot in the inventory.

**Comment by BobMowzie** [comment]
This new bug you've reported is incorrect. The axe cannot use its ability because the weapon cooldown is reset when you break a block. Unfortunately, there seems to be a bug with vanilla MC that does not display this cooldown after breaking a block.
You simply need to wait for the brief cooldown to end.

**Comment by BobMowzie** [comment]
As for your main issue, I still cannot tell what I am looking at in that screenshot. Is that your player skin? Is that another mob? A video would really help.
